### PR TITLE
Add endpoint to create new stands

### PR DESCRIPTION
**Pull request by AxonC**

### CODE REVIEW

**Comment by AxonC** [comment]
Waiting on #442 and pending changes to the Stand model from Andy.

File: app/Http/Requests/StandCreateRequest.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'identifier' => 'required|string',
```
Comment (AndyTWF) [inline app/Http/Requests/StandCreateRequest.php:27]: If you're able to give https://github.com/VATSIM-UK/uk-controller-api/pull/441 a quick review, we can add `assignment_priority` here as a positive integer?

**Review by AndyTWF — CHANGES_REQUESTED** [review]

File: app/Http/Requests/StandCreateRequest.php (hunk 1 above)
Comment (AxonC) [reply app/Http/Requests/StandCreateRequest.php:27]: Reviewed, can you merge before I push my changes?

Comment (AxonC) [reply app/Http/Requests/StandCreateRequest.php:27]: Also, is it optional?

Comment (AndyTWF) [reply app/Http/Requests/StandCreateRequest.php:27]: I've merged now - the default is 100 on that field, so it is optional in the DB.

File: app/Http/Controllers/Admin/StandAdminController.php [hunk 2]
```diff
@@ -70,4 +72,39 @@ public function getStandDetails(Airfield $airfield, Stand $stand): JsonResponse
...
+     *
+     * @param Airfield $airfield
+     * @param StandCreateRequest $request
+     * @return JsonResponse
+     */
+    public function createNewStand(Airfield $airfield, StandCreateRequest $request): JsonResponse
+    {
+        $validatorsInUse = $airfield->stands->pluck('identifier');
+        if ($validatorsInUse->contains($request->get('identifier'))) {
+            return response()->json(['message' => 'Stand identifier in use for airfield.'], 400);
```
[the remark this answers is not in the document]
Comment (AxonC) [reply app/Http/Controllers/Admin/StandAdminController.php]: Changed

File: app/Http/Requests/StandCreateRequest.php (hunk 1 above)
Comment (AxonC) [reply app/Http/Requests/StandCreateRequest.php:27]: It didn't allow a NULL, so I added a default on the controller model create. https://github.com/VATSIM-UK/uk-controller-api/pull/443/commits/cbcff762530e2743e534cae9739722c5d046bee0#diff-ea33c1e1edc02e130249c2cb106990430cc2389c4c0fc20c6b8d10e33b2708f1R108

**Review by AndyTWF — APPROVED** [review]

**Pull request merged**